src/Firestore/PathTrait.php

```diff
@@ -72,6 +77,43 @@ private function databaseFromName($name)
         return $parts[0] . '/databases/' . explode('/', $parts[1])[0];
     }
 
+    /**
+     * Return the database ID from a fully-qualified path name.
+     *
+     * This method returns a bare database ID. For the fully-qualified database
+     * name, use {@see Google\Cloud\Firestore\PathTrait::databaseFromName()}.
+     *
+     * @param string $name
+     * @return string
+     */
+    private function databaseIdFromName($name)
+    {
+        $parts = explode('/databases/', $name);
+        if (count($parts) !== 2) {
```

Comment:
What if a user names its collections 'databases'? I think you can just split by slash here and everywhere and use the well-known segment index for each lookup.